Geometry/HGCalCommonData/plugins/dd4hep/DDHGCalCell.cc

```diff
@@ -2,6 +2,7 @@
 #include "DetectorDescription/DDCMS/interface/DDPlugins.h"
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include "FWCore/Utilities/interface/Exception.h"
+#include <DD4hep/DD4hepUnits.h>
```

Comment:
I don't think the DD4hepUnits.h is needed in this file.

Reply:
Right. Forgot to remove this. It is done now